src/pages/home/sidebar/ChatSwitcherOptionPropTypes.js

```diff
@@ -10,9 +10,15 @@ export default PropTypes.shape({
     // The URL of the person's avatar
     icon: PropTypes.string,
 
+    // URLS OF FIRST TWO PARTICIPANTS' AVATAR FOR GROUP CHAT
```

Comment:
same as before, no need to make comments all caps, I would keep with the style that is already present.

Reply:
Alright. Fix coming up in my next commit.